### PR TITLE
chore(*): docker read-only mode docs

### PR DESCRIPTION
**Pull request by samugi**
### Description

Document how to start kong in Docker in `read-only` mode

### Testing instructions

Netlify link: 

### Checklist 

- [X] Review label added 
- [X] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

[KAG-852]

[KAG-852]: https://konghq.atlassian.net/browse/KAG-852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/_src/gateway/install/docker/index.md [hunk 1]
```diff
@@ -439,6 +439,63 @@ docker container rm kong-dbless
 docker network rm kong-net
 ```
 
+## Running Kong in read-only mode
+
+To run Kong in read-only mode, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path, as provided by the following example:
```
Comment (kikito) [inline app/_src/gateway/install/docker/index.md:444]: This needs a mention of "starting in 3.2.0". We offer no guarantees for previous versions.

**Review by kikito — CHANGES_REQUESTED** [review]
missing version ingo

File: app/_src/gateway/install/docker/index.md (hunk 1 above)
Comment (samugi) [reply app/_src/gateway/install/docker/index.md]: done. @Kong/docs-maintainers should this be handled with some special version tagging?

**Comment by Hayk-S** [comment]
The provided configuration looks correct and I was able to reproduce the read-only issue with this configuration.
Just FYI, in the docs, we provide `read_only` example for `db-less` mode but it is also possible to enable that mode with the classic or hybrid gateway modes.

**Review by Hayk-S — APPROVED** [review]

File: app/_src/gateway/install/docker/index.md [hunk 2]
```diff
@@ -439,6 +439,63 @@ docker container rm kong-dbless
 docker network rm kong-net
 ```
 
+## Running Kong in read-only mode
+
+Starting with version 3.2.0 of {{site.base_gateway}} it is possible to run the container in read-only mode. To do so, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path, as provided by the following example:
```
Comment (cloudjumpercat) [inline app/_src/gateway/install/docker/index.md:444]:
```suggestion
Starting with {{site.base_gateway}} 3.2.0, you can run the container in read-only mode. To do so, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path:
```

**Review by cloudjumpercat — APPROVED** [review]
Just made a small copyedit, but everything else looks good to me! Thanks for making this update!

**Pull request merged**